packages/data/addon/mirage/routes/bard-lite.js

```diff
@@ -137,10 +167,15 @@ export default function(
         if (currFilter.length > 0) {
           if (currFilter[0] === ',') currFilter = currFilter.substring(1);
 
-          let dimension = currFilter.split('|')[0],
-            values = currFilter.split('[')[1].split(',');
+          const columnOp = currFilter.split('-');
+          const column = columnOp[0].split('|');
+          const op = columnOp[1].split('[');
+          let dimension = column[0],
```

Comment:
`const` all the things

Reply:
giving up on `const` in order to use regex extraction and make small fixups to `field`and `values`